### PR TITLE
putting Tangram map on playground

### PR DESCRIPTION
**Pull request by hanbyul-here**
- putting Tangram map on playground
- Style is not decided, just using Toner for temp.  Do not merge this yet.

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -29,13 +29,13 @@
 
   
 
-  <script src="js/lib/leaflet.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.3/leaflet.js"></script>
+  <script src="https://mapzen.com/tangram/0.1/tangram.min.js"></script>
```
Comment (bcamper) [inline _layouts/default.html:33]: Use Tangram 0.3 please

File: _layouts/default.html [hunk 2]
```diff
@@ -21,7 +21,7 @@
   <link rel="icon" href="/images/favicon.png">
   <link href='//fonts.googleapis.com/css?family=Lato:400,300,700' rel='stylesheet' type='text/css'>
   <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
-  <link href="css/leaflet.css" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.3/leaflet.css" />
```
Comment (bcamper) [inline _layouts/default.html:24]: Consider using Leaflet 1.0? it has continuous zoom. Otherwise 0.7 should be OK.

File: js/views/map.js [hunk 3]
```diff
@@ -22,18 +22,41 @@ DeveloperPlayground.MapView = Backbone.View.extend({
         this.stopListening();
     },
 
-    
+
+    hasWebGL: function() {
+        //detect webgl is available on the browser
+        var canvas = document.createElement('canvas'),
+            gl,
+            glExperimental = false;
```
Comment (bcamper) [inline js/views/map.js:30]: extraneous line

File: js/views/map.js [hunk 4]
```diff
@@ -22,18 +22,41 @@ DeveloperPlayground.MapView = Backbone.View.extend({
...
-        L.tileLayer('https://{s}.tiles.mapbox.com/v3/randyme.li1lhlf0/{z}/{x}/{y}.png', {
-            maxZoom: 18,
-            attribution: 'Map data &copy; <a target="_blank" href="http://openstreetmap.org">OpenStreetMap</a> contributors, <a target="_blank" href="http://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, Imagery © <a target="_blank" href="http://mapbox.com">Mapbox</a>',
-        })
-        .addTo(this.map);
+
+        var layer;
+        if(this.hasWebGL()) {
+            var layer = Tangram.leafletLayer({
+                 scene: 'https://cdn.rawgit.com/tangrams/multiverse/gh-pages/styles/tangram-toner.yaml',
```
Comment (bcamper) [inline js/views/map.js:50]: Don't link to this style! Use the official [Refill](https://github.com/tangrams/refill) style, linked here: https://raw.githubusercontent.com/tangrams/refill/gh-pages/refill.yaml

Comment (hanbyul-here) [inline js/views/map.js:50]: I tried putting refill first, but it didn't look best with Playground. (attaching screenshot)
Waiting for Cartography team's opinion about base map.
- Playground will probably stick with raster tiles for now 

screen shot 2015-10-08 at 4 30 19 pm

Comment (bcamper) [inline js/views/map.js:50]: Ok, to be clear, Refill is the same style as tangram-toner, but more
complete (tangram-toner was the original name).

**Comment by meghanhade** [comment]
We're going to use stamen raster tiles for now. Will revisit soon though.

**Pull request closed without merging**